src/main/java/htsjdk/tribble/util/HTTPHelper.java

```diff
@@ -90,24 +90,6 @@ public InputStream openInputStream() throws IOException {
         return new WrapperInputStream(connection, connection.getInputStream());
     }
 
-
-    /**
-     * Open an input stream for the requested byte range.  Its the client's responsibility to close the stream.
-     *
-     * @param start start of range in bytes
-     * @param end   end of range ni bytes
-     * @return
-     * @throws IOException
-     */
-    @Deprecated
-    public InputStream openInputStreamForRange(long start, long end) throws IOException {
-
-        HttpURLConnection connection = openConnection();
-        String byteRange = "bytes=" + start + "-" + end;
-        connection.setRequestProperty("Range", byteRange);
-        return new WrapperInputStream(connection, connection.getInputStream());
-    }
```

Comment:
It looks save to delete,  IGV doesn't call that method.    It does use openInputStreamForPath,  but that can probably be factored out as well.    Its been many years but I think the idea of that helper class was to allow clients to supply, in essence, a factory object to create custom streams.   Specifically there was a period when IGV used the Apache http library.  

An editorial comment on the title of this issue and reference to "Unused".   Unused by whom?   All public methods of htsjdk are potentially used.